### PR TITLE
semantic_text: Updated copy-to docs

### PR DESCRIPTION
**Pull request by carlosdelest**
Make explicit what users will need to update for both bulk requests and ingest pipelines

image

### CODE REVIEW

File: docs/reference/mapping/types/semantic-text.asciidoc [hunk 1]
```diff
@@ -137,8 +137,42 @@ field to collect the values of other fields for semantic search. Each value has
...
+                "type": "text",
+                "copy_to": "infer_field"
+            }
+        }
+    }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+
+Will need the following bulk request update to ensure that `infer_field` is updated correctly:
```
Comment (Mikep86) [inline docs/reference/mapping/types/semantic-text.asciidoc:165]:
```suggestion
Will need the following bulk update request to ensure that `infer_field` is updated correctly:
```

File: docs/reference/mapping/types/semantic-text.asciidoc [hunk 2]
```diff
@@ -148,3 +182,5 @@ In bulk requests, all fields that are copied to a `semantic_text` field must hav
 
 * `semantic_text` fields are not currently supported as elements of <<nested,nested fields>>.
 * `semantic_text` fields can't be defined as <<multi-fields,multi-fields>> of another field, nor can they contain other fields as multi-fields.
+* When ingestion pipelines update an already existing `semantic_text` field value, inference won't be recalculated for the field automatically.
+The inference will only be recalculated when the document is reindexed.
```
Comment (Mikep86) [inline docs/reference/mapping/types/semantic-text.asciidoc:186]: This isn't the problem. If an ingestion pipeline updates a `semantic_text` field directly, everything is fine. The issue is if an ingestion pipeline updates a field that is copied to a `semantic_text` field. In this case, the update is silently ignored and the indexed embedding becomes stale.

Comment (carlosdelest) [reply docs/reference/mapping/types/semantic-text.asciidoc:186]: Thanks for noticing this - I started updating the limitations but decided to include them in copy_to . Forgot to clean this up! Doing it now

**Review by Mikep86 — APPROVED** [review]
LGTM!

**Pull request merged**